docs/tutorials/credentials-status-update/README.md

```diff
@@ -0,0 +1,155 @@
+# Credentials Status Update Tutorial
+
+If you wish to bypass this tutorial and simply want to import the postman collection locally, copy the "Raw" [link](https://raw.githubusercontent.com/w3c-ccg/traceability-interop/main/docs/tutorials/credentials-status-update/credentials-status-update.postman_collection.json) to your clipboard and In Postman, Import -> Link -> Paste the "Raw" link. Otherwise, please continue with the rest of the tutorial.
+
+---
+
+If you have not done so already, please review and complete the previous tutorials which form the foundation for the material presented in this tutorial.
+
+1. [Authentication Tutorial](../authentication/README.md) - Obtain an OAuth2 client credentials grant token
+1. [Did Web Discovery Tutorial](../did-web-discovery/README.md) - Get Organization identifiers and VC-API service endpoint
+1. [Credentials Issue Tutorial](../credentials-issue/README.md) - Issue a verifiable credential
+1. [Credentials Verify Tutorial](../credentials-verify/README.md) - Verify a verifiable credential
+
+In this tutorial, we will be updating the status of (specifically, revoking) a previously issued and verified verifiable credential from the Credentials Issue Tutorial workflow, and then validating that is has been properly updated.
```

Comment:
```suggestion
In this tutorial, we will update the status of a verifiable credential, previously issued and verified in the **Credentials Issue Tutorial** workflow, and then validating that is has been properly updated (specifically, revoked).
```

Reply:
@rhofvendahl — What's your reason for not applying this change? (I've just added a couple more word changes, that I didn't notice last time.)